Common/SymbolRepr.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect
+{
+    /// <summary>
+    /// Public static helper class that does parsing/generation of symbol representations (options, futures)
+    /// </summary>
+    public static class SymbolRepresentation
+    {
+        /// <summary>
+        /// Function returns underlying name, expiration year, expiration month for the future contract ticker. Function detects if
+        /// the format is used either XYZH6 or XYZH16. Returns null, if parsing failed.
+        /// </summary>
+        /// <param name="ticker"></param>
+        /// <returns>Tuple containing 1) underlying name, 2) short expiration year, 3) expiration month</returns>
+        public static Tuple<string, int, int> ParseFutureTicker(string ticker)
```

Comment:
Public APIs should **never** use the `Tuple` type. Classes are cheap and easy to define and less confusing, especially when a `Tuple` contains multiple members of the same type, such as in this case. 

Similar comment applies to other methods in this file that return a `Tuple`.